ui/src/state/settings.ts

```diff
@@ -161,8 +176,39 @@ export const useSettingsState = createState<BaseSettingsState>(
     },
     loaded: false,
     putEntry: async (bucket, key, val) => {
-      const poke = doPutEntry(window.desk, bucket, key, val);
-      await pokeOptimisticallyN(useSettingsState, poke, reduceUpdate);
+      await api.trackedPoke<PutEntry, SettingsEvent>(
+        {
+          app: 'settings-store',
+          mark: 'settings-event',
+          json: {
+            'put-entry': {
+              desk: window.desk,
+              'bucket-key': bucket,
+              'entry-key': key,
+              value: val,
+            },
+          },
+        },
+        {
+          app: 'settings-store',
+          path: `/desk/${window.desk}`,
+        },
+        (event) => {
```

Comment:
I think you may be able to omit this checking function and it will still work since the subscription already has the reduce and the default checker on tracked poke does a deep equal